bundles/automation/org.eclipse.smarthome.automation.rest/META-INF/MANIFEST.MF

```diff
@@ -5,7 +5,8 @@ Bundle-SymbolicName: org.eclipse.smarthome.automation.rest
 Bundle-Version: 0.8.0.qualifier
 Bundle-Vendor: Eclipse.org/SmartHome
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
-Import-Package: javax.ws.rs,
+Import-Package: io.swagger.annotations,
```

Comment:
We could declare this as an optional dependency, so that you are free to NOT provide the annotations at runtime.